packages/react-scripts/scripts/start.js

```diff
@@ -60,6 +60,11 @@ if (isSmokeTest) {
   };
 }
 
+// Used to detect flow errors we want to swallow
+function isFlowIntegrationErrorMessage(message) {
```

Comment:
Ideally this kind of stuff should live in `react-dev-utils` so all hacks are hidden in its implementation